pkg/skaffold/bazel/bazel.go

```diff
@@ -65,6 +66,8 @@ func GetDependencies(ctx context.Context, workspace string, a *latest.BazelArtif
 		deps = append(deps, "WORKSPACE")
 	}
 
+	logrus.Debugf("Found dependencies for bazel artifact: %s", deps)
```

Comment:
Should you use `%v` not a slice?

Reply:
it actually prints the exact same thing either way, but you're right `%v` is more correct. fixed